Quickstart has an empty directory with no files, throwing a `not found` error

When running the bazel test command inside the `nativelink` main repo, I'm getting a not found error:

{ code: NotFound, messages: ["No such file or directory (os error 2)", "Failed to remove file \"/tmp/nativelink/data-worker-test/tmp_path-cas/36517273a9afe4dace58ea5e99842a8a638131b3bf3e33c56216000000000000-12441\""] }

Running on:
bazel 7.0.2-homebrew

Slack chat: https://nativelink.slack.com/archives/C06758HJQF8/p1709776707727789


Main Idea for failure: 
The remote execution part is failing. If you do remove the --remote_executor=grpc://127.0.0.1:50051 \ line, it will correctly compile using just the remote_cache

Adding on to this I just ran through this on my mac and got this output

```
    Finished release [optimized] target(s) in 29.84s
INFO: Analyzed 59 targets (445 packages loaded, 17164 targets configured).
INFO: From Compiling utf8_validity.cc [for tool]:
external/protobuf~23.1~non_module_deps~utf8_range/utf8_validity.cc:139:12: warning: unused function 'CodepointSkipBackwards' [-Wunused-function]
inline int CodepointSkipBackwards(int32_t codepoint_word) {
           ^
1 warning generated.
INFO: From Linking external/protobuf~23.1/src/google/protobuf/io/libio_win32.a [for tool]:
warning: /Library/Developer/CommandLineTools/usr/bin/libtool: archive library: bazel-out/darwin_arm64-opt-exec-ST-13d3ddad9198/bin/external/protobuf~23.1/src/google/protobuf/io/libio_win32.a the table of contents is empty (no object file members in the library define global symbols)
ERROR: /Users/brianclark/tracemachina/nativelink/nativelink-store/BUILD.bazel:71:16: Rustfmt nativelink-store/integration_tests/fast_slow_store_test.rustfmt.ok failed: (Exit 1): process_wrapper failed: error executing Rustfmt command (from target //nativelink-store:integration_tests/fast_slow_store_test) bazel-out/darwin_arm64-opt-exec-ST-13d3ddad9198/bin/external/rules_rust~0.40.0/util/process_wrapper/process_wrapper --touch-file ... (remaining 9 arguments skipped)
dyld[73574]: Library not loaded: @rpath/librustc_driver-132ebc126ee115d9.dylib
  Referenced from: <DBF3D0E7-C26F-31E4-BADB-3469FB616452> /private/tmp/nativelink/work/50148a228c0b3d621d5e8a3f77bbd8508bdaa23418c07851801580df0154184e/bazel-out/darwin_arm64-opt-exec-ST-13d3ddad9198/bin/external/rules_rust~0.40.0~rust~rust_darwin_aarch64__aarch64-apple-darwin__nightly_tools/rust_toolchain/bin/rustfmt
  Reason: tried: '/private/tmp/nativelink/work/50148a228c0b3d621d5e8a3f77bbd8508bdaa23418c07851801580df0154184e/bazel-out/darwin_arm64-opt-exec-ST-13d3ddad9198/bin/external/rules_rust~0.40.0~rust~rust_darwin_aarch64__aarch64-apple-darwin__nightly_tools/rust_toolchain/bin/../lib/librustc_driver-132ebc126ee115d9.dylib' (no such file), '/private/tmp/nativelink/work/50148a228c0b3d621d5e8a3f77bbd8508bdaa23418c07851801580df0154184e/bazel-out/darwin_arm64-opt-exec-ST-13d3ddad9198/bin/external/rules_rust~0.40.0~rust~rust_darwin_aarch64__aarch64-apple-darwin__nightly_tools/rust_toolchain/bin/../lib/librustc_driver-132ebc126ee115d9.dylib' (no such file)
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 117.016s, Critical Path: 36.98s
INFO: 946 processes: 638 internal, 308 remote.
ERROR: Build did NOT complete successfully
//nativelink-config:doc_test                                          NO STATUS
//nativelink-proto:update_protos_test                                 NO STATUS
//nativelink-scheduler:doc_test                                       NO STATUS
//nativelink-scheduler:integration_tests/action_messages_test         NO STATUS
//nativelink-scheduler:integration_tests/cache_lookup_scheduler_test  NO STATUS
//nativelink-scheduler:integration_tests/property_modifier_scheduler_test NO STATUS
//nativelink-scheduler:integration_tests/simple_scheduler_test        NO STATUS
//nativelink-service:doc_test                                         NO STATUS
//nativelink-service:integration_tests/ac_server_test                 NO STATUS
//nativelink-service:integration_tests/bytestream_server_test         NO STATUS
//nativelink-service:integration_tests/cas_server_test                NO STATUS
//nativelink-service:integration_tests/worker_api_server_test         NO STATUS
//nativelink-store:doc_test                                           NO STATUS
//nativelink-store:integration_tests/ac_utils_test                    NO STATUS
//nativelink-store:integration_tests/completeness_checking_store_test NO STATUS
//nativelink-store:integration_tests/compression_store_test           NO STATUS
//nativelink-store:integration_tests/dedup_store_test                 NO STATUS
//nativelink-store:integration_tests/existence_store_test             NO STATUS
//nativelink-store:integration_tests/fast_slow_store_test             NO STATUS
//nativelink-store:integration_tests/filesystem_store_test            NO STATUS
//nativelink-store:integration_tests/memory_store_test                NO STATUS
//nativelink-store:integration_tests/ref_store_test                   NO STATUS
//nativelink-store:integration_tests/s3_store_test                    NO STATUS
//nativelink-store:integration_tests/shard_store_test                 NO STATUS
//nativelink-store:integration_tests/size_partitioning_store_test     NO STATUS
//nativelink-store:integration_tests/verify_store_test                NO STATUS
//nativelink-util:doc_test                                            NO STATUS
//nativelink-util:integration_tests/buf_channel_test                  NO STATUS
//nativelink-util:integration_tests/evicting_map_test                 NO STATUS
//nativelink-util:integration_tests/fastcdc_test                      NO STATUS
//nativelink-util:integration_tests/fs_test                           NO STATUS
//nativelink-util:integration_tests/health_utils_test                 NO STATUS
//nativelink-util:integration_tests/resource_info_test                NO STATUS
//nativelink-util:integration_tests/retry_test                        NO STATUS
//nativelink-worker:doc_test                                          NO STATUS
//nativelink-worker:integration_tests/local_worker_test               NO STATUS
//nativelink-worker:integration_tests/running_actions_manager_test    NO STATUS
//:dummy_test                                                            PASSED in 0.2s

Executed 1 out of 38 tests: 1 test passes and 37 were skipped.
```

--- a/.github/styles/config/vocabularies/TraceMachina/accept.txt
+++ b/.github/styles/config/vocabularies/TraceMachina/accept.txt
@@ -24,3 +24,4 @@ blazingly
 mutex
 parsable
 rebase
+remoteable

--- a/README.md
+++ b/README.md
@@ -14,6 +14,11 @@ Supports Unix-based operating systems and Windows.
 
 Below, you will find a few different options for getting started with NativeLink.
 
+### 📝 Clone the NativeLink repository
+1. Go to the [NativeLink](https://github.com/TraceMachina/nativelink) repository on GitHub. Clone the repository via SSH or HTTPS. In this example the repository is cloned via SSH:
+```bash
+git clone git@github.com:TraceMachina/nativelink.git
+```
 
 ### 📦 Installing with Cargo
 
@@ -43,10 +48,12 @@ nativelink basic_cas.json
 
 ## 🧪 Evaluating NativeLink
 
-Once you've built NativeLink and have an instance running with the
-`basic_cas.json` configuration, launch a separate terminal session and run the
-following command to connect the running server launched above to Bazel or
-another RBE client:
+1. Once you've built NativeLink and have an instance running with the `basic_cas.json` configuration, launch a separate terminal session.
+2. Navigate to where you cloned the NativeLink repository:
+```bash
+cd $HOME/nativelink
+```
+3. In the new terminal, run the following command to connect the running server launched above to Bazel or another RBE client:
 
 ```sh
 bazel test //... \
@@ -67,6 +74,9 @@ bazel test //... `
 ```
 This causes Bazel to run the commands through an all-in-one `CAS`, `scheduler`
 and `worker`.
+
+> [!WARNING]
+> If you're using MacOS, encountering errors is anticipated at this stage. Our team is actively working on enhancing support for executing remoteable Bazel builds with MacOS. For now, you can run with [Docker](https://github.com/blakehatch/nativelink/tree/main/deployment-examples/docker-compose) or a Linux virtual machine. If you have any questions, reach out on the [NativeLink](https://join.slack.com/t/nativelink/shared_invite/zt-2forhp5n9-L7dTD21nCSY9_IRteQvZmw) slack.
 
 ## How it Works
 

